scripts/update-homebrew.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Pushes the latest Yarn version to Homebrew
+
+set -ex
+
+version=`curl --fail https://yarnpkg.com/latest-version`
+
+# Ensure Linuxbrew is on the PATH
+PATH=$PATH:$HOME/.linuxbrew/bin/
+# Ensure homebrew-core is pointing to Homebrew rather than Linuxbrew
+pushd ~/.linuxbrew/Library/Taps/homebrew/homebrew-core
+#git remote set-url origin https://github.com/Daniel15/homebrew-core # for testing
+git remote set-url origin https://github.com/homebrew/homebrew-core
+# Remove any existing branch (eg. if the previous attempt failed)
+git branch -D yarn-$version || true
```

Comment:
Good point. It should always be on `master` when the script is ran, as `bump-formula-pr` switches to `master` after pushing the PR. I'll probably change it in a followup PR but I think this is okay for now.